### ISSUE TITLE
DotvvmStartup.ConfigureServices does not get called in ASP.NET Core project generated by Visual Studio extension

### ISSUE THREAD

**Issue by vstribrny** [riganti/dotvvm#636]
After the ASP.NET Core project is  generated by DotVVM Visual Studio extension, the `DotvvmStartup.ConfigureServices` method does not get called. In that method, there is even a registration statement `options.AddBusinessPack();` when I choose to add Business Pack controls in the VS extension's dialog box.

I guess we should call overload of `services.AddDotVVM` in Startup.ConfigureServices with DotvvmStartup for TServiceConfigurator generic parameter since DotvvmStartup implements IDotvvmServiceConfigurator:

`services.AddDotVVM<DotvvmStartup>(); // works fine`

**Comment by adamjez** [comment]
What version of Visual Studio extension do you have? 


I've tested it and I wasn't able to reproduce it.

**Comment by vstribrny** [comment]
The version is 2.0.115.0, it's the latest I think. I tried it again, chose "DotVVM Business Pack". It behaves the same as described above. As a result, when I want to use some BP control, it throws with

> The control <bp:TextBox> could not be resolved! Make sure that the tagPrefix is registered in DotvvmConfiguration.Markup.Controls collection!
1: @viewModel DotvvmApplication2.ViewModels.DefaultViewModel, DotvvmApplication2

It's logical since `options.AddBusinessPack();` did not get called... After I changed code in `Startup.ConfigureServices` as I described above, it starts to work.
